sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/ShareAsyncAPITests.groovy

```diff
@@ -190,9 +193,20 @@ class ShareAsyncAPITests extends APISpec {
             }
     }
 
+    @Unroll
     def "Get properties premium"() {
         given:
-        def premiumShare = premiumFileServiceAsyncClient.createShareWithResponse(generateShareName(), testMetadata, null).block().getValue()
+        ShareEnabledProtocol enabledProtocol = new ShareEnabledProtocol()
+        if (Constants.HeaderConstants.SMB_PROTOCOL == protocol) {
```

Comment:
Same comment as above

Reply:
Yea! I added parse after these tests, so I didn't think of that. Good catch!